Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -828,16 +828,14 @@ extension CarPlayManager: CPMapTemplateDelegate {
         delegate?.carPlayManagerDidEndNavigation(self, byCanceling: false)
     }
     
+    public func mapTemplateDidBeginPanGesture(_ mapTemplate: CPMapTemplate) {
+        // Whenever panning starts - stop any navigation camera updates.
+        navigationMapView?.navigationCamera.stop()
+    }
+    
     public func mapTemplate(_ mapTemplate: CPMapTemplate, didEndPanGestureWithVelocity velocity: CGPoint) {
-        // We want the panning surface to have "friction". If the user did not "flick" fast/hard enough, do not update the map with a final animation.
-        guard sqrtf(Float(velocity.x * velocity.x + velocity.y * velocity.y)) > 100 else {
-            return
-        }
-        
-        let decelerationRate: CGFloat = 0.9
-        let offset = CGPoint(x: velocity.x * decelerationRate / 4,
-                             y: velocity.y * decelerationRate / 4)
-        updatePan(by: offset, mapTemplate: mapTemplate, animated: true)
+        // After panning is stopped - allow navigation bar dismissal.
+        mapTemplate.automaticallyHidesNavigationBar = true
```

Comment:
Deceleration rate usage causes several issues while panning, so I've decided to remove it. In future we can consider to add it back.

Reply:
Could you create a ticket for considering adding it back?

Reply:
Yep, will consider adding it back. Will not be part of first iteration of panning support.